src/Network/Network/Cortex/VirtualHub/VirtualHubBaseCmdlet.cs

```diff
@@ -94,6 +111,32 @@ public PSVirtualHub CreateOrUpdateVirtualHub(string resourceGroupName, string vi
             PSVirtualHub hubToReturn = this.ToPsVirtualHub(virtualHubCreatedOrUpdated);
             hubToReturn.ResourceGroupName = resourceGroupName;
 
+            if (psHubVnetConnections == null)
+            {
+                return hubToReturn;
+            }
+
+            // Crud operation on the vnet connections
+            hubToReturn.VirtualNetworkConnections = new List<PSHubVirtualNetworkConnection>();
+
+            foreach (var psConnection in psHubVnetConnections)
```

Comment:
👍

Reply:
Added change log. And changed message to *VirtualHubVnetConnection* to capture all below commands and their AzureRM counterparts-
Cmdlet          New-AzVirtualHubVnetConnection                     3.0.0      Az.Network
Cmdlet          Remove-AzVirtualHubVnetConnection                  3.0.0      Az.Network
Cmdlet          Update-AzVirtualHubVnetConnection                  3.0.0      Az.Network